components/tooltip/index.css

```diff
@@ -13,17 +13,75 @@ governing permissions and limitations under the License.
-  --spectrum-tooltip-neutral-tip-width: var(--spectrum-tooltip-neutral-tip-height);
+  --spectrum-tooltip-animation-duration: var(--spectrum-animation-duration-100);
+
+  /* animation distance is equal to spacing to source */
+  --spectrum-tooltip-animation-distance: var(--spectrum-spacing-75);
+
+  /* override if additional spacing to source is required */
+  --spectrum-tooltip-margin: 0px;
+
+  --spectrum-tooltip-height: var(--spectrum-component-height-75);
+  --spectrum-tooltip-max-inline-size: var(--spectrum-tooltip-maximum-width);
+  --spectrum-tooltip-border-radius: var(--spectrum-corner-radius-100);
+
+  --spectrum-tooltip-icon-width: var(--spectrum-workflow-icon-size-50);
+  --spectrum-tooltip-icon-height: var(--spectrum-workflow-icon-size-50);
+
+  --spectrum-tooltip-font-size: var(--spectrum-font-size-75);
+  --spectrum-tooltip-line-height: var(--spectrum-line-height-100);
+  --spectrum-tooltip-line-height-cjk: var(--spectrum-cjk-line-height-100);
+  --spectrum-tooltip-font-weight: var(--spectrum-font-weight-regular); /* TODO - check is still 400 */
+
+  /* horizontal spacing */
+  --spectrum-tooltip-spacing-inline-start: var(--spectrum-component-edge-to-visual-75);
+  --spectrum-tooltip-spacing-inline-end: var(--spectrum-component-edge-to-text-75);
+
+  /* vertical spacing */
+  --spectrum-tooltip-spacing-block-start: var(--spectrum-component-top-to-text-75);
+  --spectrum-tooltip-spacing-block-end: var(--spectrum-component-bottom-to-text-75);
+
+  /* overall margin */
+  --spectrum-tooltip-spacing-container: var(--spectrum-spacing-100);
+
+  /* icon spacing */
+  --spectrum-tooltip-icon-spacing-inline-end: var(--spectrum-text-to-visual-75);
+  --spectrum-tooltip-icon-spacing-block-start: var(--spectrum-component-top-to-workflow-icon-75);
+
+  /* colors */
+  --spectrum-tooltip-background-color-informative: var(--spectrum-informative-background-color-default);
+  --spectrum-tooltip-background-color-positive: var(--spectrum-positive-background-color-default);
+  --spectrum-tooltip-background-color-negative: var(--spectrum-negative-background-color-default);
+
+  --spectrum-tooltip-content-color: var(--spectrum-white);
+
+  /* tip */
+  --spectrum-tooltip-tip-inline-size: var(--spectrum-tooltip-tip-width);
+  --spectrum-tooltip-tip-block-size: var(--spectrum-tooltip-tip-height);
+
+  /* pointer tip - default spacing to edge - corner radius plus half of tip width to neutralize override */
+  --spectrum-tooltip-pointer-corner-spacing: var(--spectrum-corner-radius-100);
 }
 
+@media (forced-colors: active) {
+  .spectrum-Tooltip {
+    border: 1px solid transparent;
+  }
+
+  .spectrum-Tooltip-tip {
+    forced-color-adjust: none;
+    --spectrum-tooltip-background-color-default: CanvasText;
+    --spectrum-tooltip-background-color-informative: CanvasText;
+    --spectrum-tooltip-background-color-positive: CanvasText;
+    --spectrum-tooltip-background-color-negative: CanvasText;
```

Comment:
Should these be `--highcontrast-*` prefixed custom properties?

Reply:
Yes, thank you for this catch!